src/libraries/libraries-packages.proj

```diff
@@ -16,7 +15,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(PkgDir)*\*.proj" Exclude="$(PkgDir)test\*" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)*\pkg\**\*.pkgproj" Condition="('$(BuildAllConfigurations)' == 'true' or '$(DotNetBuildFromSource)' == 'true') And '$(BuildAllOOBPackages)' == 'true'" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)*\src\**\*.*proj" Condition="('$(BuildAllConfigurations)' == 'true' or '$(DotNetBuildFromSource)' == 'true') And '$(BuildAllOOBPackages)' == 'true'" Exclude="$(MSBuildThisFileDirectory)*\src\**\*.shproj"/>
```

Comment:
@Anipik don't you want to exclude .pkgprojs here as well?

Reply:
pkgprojs wont get picked because its only looking into src\**\ which wont be true for pkgprojs

Reply:
ah right, perfect.

Reply:
Why do you look into nested src folders as well? `$(MSBuildThisFileDirectory)*\src\*.*proj` should be sufficient. The nested glob worries me that we might accidentally pack too much.